docs/help-center/using-famedly/notes.mdx

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Could you add the instructions to docs/help-center/using-famedly/messages.mdx instead of creating a new page?
Asked @rvanwaveren about it and we think it would make sense to have it there instead.